### DISCUSSION TITLE
cargo run not giving output

### DISCUSSION THREAD

**Discussion by kas12345hi** [bevyengine/bevy#10134] [Q&A] [answered]
I am trying bevy and rust for the first time. I have installed bevy, I have added the println!("Hello world!"); to my main.rs file but when I type cargo run it is not showing the output. it is only showing two lines starting with Finished dev and Running

**Comment by SIGSTACKFAULT** [comment]
Can you post your `main.rs`?

**Reply by kas12345hi** [reply]
here is my main.rs

fn main() {
    println!("hello world!");
}

**Reply by SIGSTACKFAULT** [reply, marked as answer]
did you remember to save?

**Reply by kas12345hi** [reply]
I have saved the file and tried again it works now. Thankyou very much!
